<launch>
    <node name="alarm_sever" pkg="ros_alarms" type="alarm_server.py" output="screen">
        <!-- A folder that is in your system path containing all alarm handlers-->
        <param name="handler_module" type="string" value="test_handlers"/>
    </node>

    <!-- Meta alarm definitions -->
    <rosparam ns="meta_alarms">
        kill: [("test1", 2), "test2", "test3"]
        another: ["testing1", "testing2", "testing3"]
    </rosparam>

    <!-- List known alarms here -->
    <rosparam ns="known_alarms">
        test1, test2, test3
    </rosparam>
</launch>
